### DISCUSSION TITLE
WOCHE 08 - RUP SAD und Diagramme

### DISCUSSION THREAD

**Discussion by niklaskrauth** [mausio/corgi-shop-doc#10] [Blogs]
Diese Woche wurde an dem [RUP SAD](https://github.com/mausio/corgi-shop-doc/blob/main/rup_sad/RUP_SAD_Document.md) gearbeitet welches zu einem späteren Zeitpunkten erweitert wird. 

Dazu sind hier alle bisherigen Diagramme:

1. Sequenzdiagramm: 
![sqd_4 drawio](https://github.com/mausio/corgi-shop-doc/assets/122524882/e48f0105-1bd5-4912-8f99-61b5dcc6eb15)

2. Use-Case-Diagramm:
![ucd1](https://github.com/mausio/corgi-shop-doc/assets/122524882/3a5d41ba-8358-4f5e-aacd-e3df6feb296a)

3. Aktivitätsdiagramm:
User logs

<img width="216" alt="122Bildschirmfoto_2023-11-09_um_11 49 45" src="https://github.com/mausio/corgi-shop-doc/assets/122524882/8b342eee-2896-40f4-be66-c79c16554441">

Get corgis from backend

<img width="252" alt="121" src="https://github.com/mausio/corgi-shop-doc/assets/122524882/8f0f5fec-86a8-4c91-aeae-0d8192fcf518">

Get cart from backend

<img width="171" alt="12233Bildschirmfoto_2023-11-09_um_11 48 02" src="https://github.com/mausio/corgi-shop-doc/assets/122524882/5947734b-c099-4c3f-9132-9f3a65d9cc05">

4. Klassendiagramm:
![Klassendiagramm](https://github.com/mausio/corgi-shop-doc/assets/115564658/34a95ef7-ed94-4cdd-be50-780879b359b9)

5. Komponentendiagramm:
![KD_1 drawio](https://github.com/mausio/corgi-shop-doc/assets/115564658/7a6c933a-a0aa-4574-937f-e431d426810f)

**Comment by ghost** [comment]
Das war ihr habt, sieht schon mal gut aus. Nur ein bisschen Schade, dass euer SAD noch nicht fertig ist, bzw. auch zu großen Teilen nicht gefüllt ist. Als kleine Anmerkung noch, wäre es schön gewesen, wenn euer Use-Case-D. im selben Stil wäre, wie alle anderen Diagramme. Denn das ist das Einzige, welches nicht in dem selben Stil ist.

**Comment by JannisFuchs** [comment]
Das was da ist sieht schonmal gut aus, allerdings fehlen noch Teile der Aufgaben. Die Diagramme sind gut lesbar und sehen grafisch ansprechend aus.

**Comment by Puggingtons** [comment]
Mir gefallen eure Diagramme, jedoch sind beim Klassendiagramm 2 Kanten (unten links und oben mittig) ohne Verbindung :/
